instrumentation/dubbo/pom.xml

```diff
@@ -26,8 +26,8 @@
 
   <properties>
     <main.basedir>${project.basedir}/../..</main.basedir>
-    <main.java.version>1.6</main.java.version>
-    <main.signature.artifact>java16</main.signature.artifact>
+    <main.java.version>1.8</main.java.version>
```

Comment:
is dubbo itself minimum java 1.8 now? please check

Reply:
Nevermind I checked https://github.com/apache/dubbo/commit/6cab4ef1777f2f9d11d70848db2d3d4445603fa6